components/ui/theme-icon.tsx

```
@@ -1,8 +1,6 @@
 import * as React from "react"
 import { useTheme } from "next-themes"
-
-import Moon from "@/components/svgs/moon.svg"
-import Sun from "@/components/svgs/sun.svg"
+import { Moon, Sun } from "lucide-react"
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)